src/views/AssignPickerModal.vue

```diff
@@ -0,0 +1,211 @@
+      <div>
+        <ion-radio-group v-model="selectedPicker">
+          <ion-item v-for="(picker, index) in availablePickers" :key="index">
+            <ion-label>{{ picker.name }}</ion-label>
+            <ion-radio slot="end" :value="picker.id" ></ion-radio>
+          </ion-item>
+        </ion-radio-group>
+      </div>
+    </ion-list>
+    <ion-infinite-scroll
+      @ionInfinite="loadMorePickers($event)"
+      threshold="100px"
+      :disabled="!isScrollable"
+    >
+      <ion-infinite-scroll-content
+        loading-spinner="crescent"
+        :loading-text="$t('Loading')"
+      />
+    </ion-infinite-scroll>
+  </ion-content>
+</template>
+
+<script>
+import { 
+  IonButtons,
+  IonButton,
+  IonContent,
+  IonHeader,
+  IonIcon,
+  IonItem,
+  IonLabel,
+  IonList,
+  IonListHeader,
+  IonRadio,
+  IonRadioGroup,
+  IonSearchbar,
+  IonTitle,
+  IonToolbar,
+  IonInfiniteScroll,
+  IonInfiniteScrollContent,
+  modalController } from "@ionic/vue";
+import { defineComponent } from "vue";
+import { closeOutline } from "ionicons/icons";
+import { useStore } from "vuex";
+import { hasError, showToast } from "@/utils";
+import { translate } from "@/i18n";
+import { PicklistService } from '@/services/PicklistService'
+
+export default defineComponent({
+  name: "AssignPickerModal",
+  components: { 
+    IonButtons,
+    IonButton,
+    IonContent,
+    IonHeader,
+    IonIcon,
+    IonItem,
+    IonLabel,
+    IonList,
+    IonListHeader,
+    IonRadio,
+    IonRadioGroup,
+    IonSearchbar,
+    IonTitle,
+    IonToolbar,
+    IonInfiniteScroll,
+    IonInfiniteScrollContent
+  },
+  props: ['order'],
+  data () {
+    return {
+      selectedPicker: '',
+      queryString: '',
+      availablePickers: [],
+      isScrollable: true
+    }
+  },
+  methods: {
+    closeModal() {
+      modalController.dismiss({ dismissed: true });
+    },
+    async searchPicker () {
+      this.availablePickers = []
+      this.fetchPickers()
+    },
+    readyForPickup () {
+      if (this.selectedPicker.length) {
+        this.store.dispatch('picklist/createPicklist', { order: this.order, selectedPicker: this.selectedPicker })
+        modalController.dismiss({ dismissed: true });
+      } else {
+        showToast(translate('Select a picker'))
+      }
+    },
+    async loadMorePickers(event) {
+      this.fetchPickers(
+        undefined,
+        Math.ceil(
+          this.availablePickers.length / (process.env.VUE_APP_VIEW_SIZE)
+        ).toString()
+      ).then(() => {
+        event.target.complete();
+      });
+    },
+    async fetchPickers(vSize, vIndex) {
+      let inputFields = {}
+
+      if(this.queryString.length > 0) {
+        // TODO: having issue when creating more than 2 groups in performFind, searching always work on first two groups
+        // For now enabled searching on first name and externalId
+        inputFields = {
+          firstName_fld0_value: this.queryString,
```

Comment:
We do need `fld0` and `fld1`
```suggestion
          firstName_fld0_value: this.queryString,
```